File: FE/FE/src/interfaces/game.ts
export interface RuleDto { divisor: number; word: string; }
export interface GameDto { id: number; name: string; author: string; rules: RuleDto[]; /* ... */ }
export interface CreateGameDto { name: string; author: string; minRange: number; maxRange: number; rules: RuleDto[]; }
export interface UpdateGameDto extends CreateGameDto {}

File: BE/Domain/Entities/Game.cs
namespace BE.Domain.Entities
{
    public class Game
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Author { get; set; }
        public int MinRange { get; set; }
        public int MaxRange { get; set; }

        public IList<Rule> Rules { get; set; } = new List<Rule>();
        public IList<Session> Sessions { get; set; } = new List<Session>();
    }
}
